### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Central Certificate Score_Keyboard: Keyboard focus is not proper when we expand "Select Folder" button.First keyboard is going the "New Root" button instead of going to the "Refresh button.

### ISSUE THREAD

**Issue by Accessibility56** [Microsoft/IIS.WebManager#114]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #.NETWebsites; #Keyboard; #WCAG2.4.3; #C+E-H; #Chrome; #Edge

**Environment Details:**
**URL:** https://manage.iis.net/webserver/central-certificate-store
**Browser Details:** Edge(42.17134.1.0),Chrome(68.0.3440.106)
Keyboard

**Repro Steps:**
1.Download and install "Microsoft IIS Administration" application.
2.Enable Internet information Services.
3.Open "Microsoft IIS" application using the URL: https://manage.iis.net/.
4.Tab till Web Server and press enter.
5.Tab till Central Certificate Store and press enter.
6.Tab till "Select Folder" button and press enter.
7.Verify focus order inside "Select Folder" button.

**Actual Result:**
Keyboard focus is not proper when we expand "Select Folder" button. First keyboard is going the "New Root" button instead of going to the "Refresh button.

**Note**: Focus order is not proper at "Physical path" field also. Focus is first to the "Select Folder" instead of going to the "Physical path" first.

**Expected Result:**
Keyboard focus should go to the "Refresh" button first instead of going to the "New root" button. Keyboard focus should be from top to bottom and left to right.

**User Impact:** 
Keyboard only users find difficulty if keyboard focus order is not as expected.

**Attachment:**
[Bug_114_Focus order issue.pptx](https://github.com/Microsoft/IIS.WebManager/files/2326732/Bug_114_Focus.order.issue.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev2;

**Comment by sspatil123** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will  be removed.
